### PR TITLE
Use feature flag on the server side.

### PR DESCRIPTION
**Pull request by idealisms**
This change adds a router for retro certs with a single
placeholder post handler. We only add the retro-certs
router if we're in dev mode or if the env variable
ENABLE_RETRO_CERTS=1 is set.

I moved the test file since it's really testing app.js.

To test the post handler, I ran:
  curl -d 'postdata' http://localhost:3000/retroactive-certification/save-data
With the feature enabled or disabled.

===

Resolves #XXX

- [ ] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [X] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: src/routes/single-page-app.test.js [hunk 1]
```diff
@@ -1,43 +0,0 @@
-import { init } from "../app";
```
Comment (sharonwarner) [inline src/routes/single-page-app.test.js:1]: Was this change on purpose?

Comment (idealisms) [reply src/routes/single-page-app.test.js:1]: Yes, git just didn't understand that it's a file move. I was moving this file to src/app.test.js since the test imports app and then runs some tests.
